src/components/MessagesList/MessagesGroup/Message/MessagePart/FilePreview.vue

```diff
@@ -337,6 +366,10 @@ export default {
 		max-height: 64px;
 	}
 
+	.mimeicon {
```

Comment:
`mimeicon` or `mime-icon`?

Reply:
mimeicon as we often use dashes for semantic separation